user/tutorial.md

```diff
@@ -14,16 +14,54 @@ instead.
 
 To start using Travis CI, make sure you have:
 
- * A [GitHub](https://github.com/) account.
- * Owner permissions for a project [hosted on GitHub](https://help.github.com/categories/importing-your-projects-to-github/).
+ * A [GitHub](https://github.com/) or [Bitbucket](https://bitbucket.org/) account.
+ * Owner permissions for a project hosted on [GitHub](https://help.github.com/categories/importing-your-projects-to-github/) or [Bitbucket](https://confluence.atlassian.com/bitbucketserver/importing-code-from-an-existing-project-776640909.html).
```

Comment:
This link https://confluence.atlassian.com/bitbucketserver/importing-code-from-an-existing-project-776640909.html is for the server version of Bitbucket. I would rather link to this more generic page https://confluence.atlassian.com/bitbucket/repository-setup-877174034.html

Reply:
Or maybe even this one https://confluence.atlassian.com/bitbucket/teams-321853005.html ?

Reply:
I'll change it to this one: https://confluence.atlassian.com/bitbucket/transfer-repository-ownership-289964397.html